src/io/flutter/android/AndroidEmulator.java

```diff
@@ -76,4 +76,19 @@ public void processTerminated(ProcessEvent event) {
       FlutterMessages.showError("Error Opening Emulator", e.toString());
     }
   }
+
+  @Override
+  public boolean equals(Object obj) {
+    if (!(obj instanceof AndroidEmulator)) {
+      return false;
+    }
+
+    final AndroidEmulator other = (AndroidEmulator)obj;
+    return other.id.equals(id);
+  }
```

Comment:
[optional]: this could be condensed to 
`return obj instanceof AndroidEmulator && (AndroidEmulator)obj.id.equals(id);`

Reply:
done